lib/mapped-device.js

```diff
@@ -113,6 +113,9 @@ module.exports.MappedDevice = class MappedDevice {
     // assumed to belong together)
     let groups = this.groupCapabilities();
 
+    // a thermostat accessory can have multiple services, thermostat service must be the main one
+    this.#maps.sort((x, y) => x.name === 'thermostat' ? -1 : y.name === 'thermostat' ? 1 : 0);
```

Comment:
Since the main service is static according a given class, I propose to proceed this way: https://github.com/robertklep/name.klep.homekitty/pull/111/commits/0e32cdab1de5ec7741f8ddb96f57d53008255b16

Priority would be useful if sorting wasn't related to a given device, but prioritizing the services among themselves is meaningless.

Indeed in case of multiple services, it's possible to prioritize them, but it would still be relative to a given device, so proceeding staticly rather than a service property would make more sense:

const servicePriority = {
  heatpump: ['thermostat', 'fan', ...],
  ...
};

Reply:
`priority` is meaningless if only related among services themselves, because it's always related to a given device, but I proposed something at the service level by creating a new map dedicated to thermostat fan – relying on what is done for other services.